net/sched/act_api.c

```diff
@@ -658,7 +659,17 @@ int tcf_action_destroy(struct tc_action *actions[], int bind)
 			module_put(ops->owner);
 		else if (ret < 0)
 			return ret;
+		else if (atomic_read(&a->ops->delayed_delete))
+			t_actions[j++] = a;
 	}
+
+	if (j)
+		for (i = 0; i <= j && t_actions[i]; i++) {
+			a = t_actions[i];
+
+			if (!atomic_read(&a->tcfa_bindcnt))
```

Comment:
Find where bindcnt is put, and check if it gets to zero there and do the delete there